WordPress/Classes/ViewRelated/Notifications/ReplyTextView/ReplyTextView.swift

```diff
@@ -210,7 +210,7 @@ import Gridicons
                 self.btnReplyPressed()
             }
 
-            //Dimiss the fullscreen view, once it has fully closed process the saving if needed
+            // Dismiss the fullscreen view, once it has fully closed process the saving if needed
```

Comment:
Bumped into this while looking for typos in "dismiss". It's just a comment, but it was useful to fix it so it didn't appear in the search.